feature/src/main/java/org/woowatechcamp/banchan/feature/home/HomeViewModel.kt

```diff
@@ -30,7 +30,11 @@ class HomeViewModel @Inject constructor(
             getCartUseCase()
                 .onSuccess {
                     it.collectLatest { cartMap ->
-                        _orderItemsInCart.value = UiState.Success(cartMap)
+                        if (cartMap.isEmpty()) {
+                            _orderItemsInCart.value = UiState.Empty
+                        } else {
+                            _orderItemsInCart.value = UiState.Success(cartMap)
+                        }
```

Comment:
```suggestion
                        _orderItemsInCart.value = if (cartMap.isEmpty()) {
                             UiState.Empty
                        } else {
                             UiState.Success(cartMap)
                        }
```
조건문이니까 이렇게 리턴으로 변수에 넣어줘도 될 것 같아요!